spec/support/shared_examples/cnab400.rb

```diff
@@ -18,7 +18,7 @@
         agencia: '12345',
         conta_corrente: '1234567',
         digito_conta: '1',
-        empresa_mae: 'asd',
+        empresa_mae: 'ASD',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.